### DISCUSSION TITLE
Get-PoshThemes does not exist

### DISCUSSION THREAD

**Discussion by OxeStone** [JanDeDobbeleer/oh-my-posh#4021] [Troubleshoot]
Windows 11 and PowerShell 7.3.5
I have installed oh-my-posh. I am running into a couple of issues:

1. Get-PoshThemes does not work. I get the following error message:
Get-PoshThemes: The term 'Get-PoshThemes' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

2. I cannot uninstall using 'Uninstall-Module'. I get the following error message:
Uninstall-Package: No match was found for the specified search criteria and module names 'OhMyPosh'.

Please advise. Thank you!

**Comment by JanDeDobbeleer** [comment]
@OxeStone how did you install? The module has been deprecated for almost 2 years now, so that could be the issue here.

**Reply by OxeStone** [reply]
From: [https://ohmyposh.dev/docs/installation/windows](https://deref-mail.com/mail/client/8-BVL-MhKNE/dereferrer/?redirectUrl=https%3A%2F%2Fohmyposh.dev%2Fdocs%2Finstallation%2Fwindows)

winget install JanDeDobbeleer.OhMyPosh -s winget

Thank you,

**Reply by JanDeDobbeleer** [reply]
@OxeStone did you follow the guide [here](https://ohmyposh.dev/docs/installation/prompt) to set up your prompt?

**Reply by OxeStone** [reply]
Yes. I have been able to apply themes for 'cmd' (using clink) and 'Windows Terminal' by following the guide you specified above.

**Reply by JanDeDobbeleer** [reply]
@OxeStone what does your PowerShell `$PROFILE` look like?

**Reply by OxeStone** [reply]
Don't have one.

**Reply by JanDeDobbeleer** [reply]
@OxeStone that means you didn't set up PowerShell and therefore do not have anything that oh-my-posh offers. Follow the guide on how to configure PowerShell and Get-PoshThemes will be available for you.

**Reply by OxeStone** [reply]
Thank you. It's working.
